packages/taquito-local-forging/src/codec.ts

```diff
@@ -243,6 +303,14 @@ export const smartContractAddressDecoder = (val: Uint8ArrayConsumer) => {
   throw new InvalidContractAddressError(val.toString());
 };
 
+export const smartRollupContractAddressDecoder = (val: Uint8ArrayConsumer) => {
```

Comment:
I don't think SRC1 ids are `Smart Rollup Contracts`. I'm pretty sure they're **Cemented** smart rollups

Reply:
ahh i just assumed that the src were for smart rollup contracts 
I'll look for a more appropriate name

Reply:
just looked 
its short for `smart_rollup_commitment_hash` will change!

Reply:
yes, thanks for looking it up. Commitment hashes are 'cemented' after a certain amount of time.

Reply:
in this case we could name it `smartRollupCommitmentHashDecoder` and the same for encoder